app/controllers/admin/submissions_controller.rb

```diff
@@ -79,7 +79,7 @@ def show
       @partner_submissions_count =
         notified_partner_submissions.group_by_day.count
       @offers = @submission.offers
-      @notes = @submission.notes
+      @notes = @submission.notes + @submission.user.notes
```

Comment:
I wonder if we should visually show if the note is about the submission or the user. I can imagine that it could look confusing to just mix all notes. Do you have a good idea how we could do that?

Reply:
We could also ask whoever requested this feature.

Reply:
@olerichter00 you could ask in the ticket. They're pretty responsive.